packages/api-reference/src/components/Content/Content.vue

```diff
@@ -198,4 +208,10 @@ const isLazy =
   ) {
   margin-top: 0;
 }
+.section-flare {
+  position: absolute;
+  top: 0;
+  right: 0;
+  pointer-events: none;
+}
```

Comment:
Do we not need the z-index anymore?

Reply:
nope because the content is inherently above it now

Reply:
🙏